syn/syn.ys

```diff
@@ -0,0 +1,18 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# yosys synthesis script
+
+read -sv prim_clock_gating.v syn_out/*.v
```

Comment:
a 'read_liberty -lib cmos_cells.lib' could be useful here.  This means that when using the show command and otherwise exploring the design yosys understands what the cells are.

Reply:
Cool, I can add it (or alternatively, if you just want to LGTM this PR and then create another one with your improvements?)